net/udp/Kconfig

```diff
@@ -50,11 +50,12 @@ config NET_UDP_CHECKSUMS
 		Enable/disable UDP checksum support.  UDP checksum support is
 		REQUIRED for IPv6.
 
-config NET_UDP_CONNS
-	int "Number of UDP sockets"
-	default 8
+config NET_UDP_CONNS_PER_ALLOC
+	int "Number of UDP sockets per block(one for each open)"
+	default 1 if DEFAULT_SMALL
+	default 4
 	---help---
-		The maximum amount of open concurrent UDP sockets
+		The amount of open concurrent UDP sockets per allocation
```

Comment:
The number of UDP sockets allocated per allocation that occurs when none are available.

Reply:
Done